development/mimir-read-write-mode/config/nginx.conf

```diff
@@ -0,0 +1,108 @@
+worker_processes  5;  ## Default: 1
+error_log  /dev/stderr error;
+pid        /tmp/nginx.pid;
+worker_rlimit_nofile 8192;
+
+events {
+  worker_connections  4096;  ## Default: 1024
+}
+
+http {
+  client_body_temp_path /tmp/client_temp;
+  proxy_temp_path       /tmp/proxy_temp_path;
+  fastcgi_temp_path     /tmp/fastcgi_temp;
+  uwsgi_temp_path       /tmp/uwsgi_temp;
+  scgi_temp_path        /tmp/scgi_temp;
+
+  default_type application/octet-stream;
+  log_format   main '$remote_addr - $remote_user [$time_local]  $status '
+        '"$request" $body_bytes_sent "$http_referer" '
+        '"$http_user_agent" "$http_x_forwarded_for"';
+  access_log   /dev/stderr  main;
+
+  sendfile     on;
+  tcp_nopush   on;
+  resolver 127.0.0.11 ipv6=off;
+
+  # Ensure that X-Scope-OrgID is always present, default to the no_auth_tenant for backwards compatibility when multi-tenancy was turned off.
+  map $http_x_scope_orgid $ensured_x_scope_orgid {
+    default $http_x_scope_orgid;
+    "" "anonymous";
+  }
+
+  server {
+    listen 9095;
```

Comment:
nit: `9095` seems like an odd choice for the reverse proxy since that's what we use internally for gRPC. Maybe `8080`?

Reply:
Good call. Will update!